scripts/installer/install.sh

```diff
@@ -5,6 +5,7 @@
 #  SPDX-License-Identifier:     GPL-2.0
 
 BISDN_LINUX_VOLUME_LABEL="BISDN-Linux"
+BISDN_LINUX_VOLUME_SIZE_MB=6144
```

Comment:
I wonder if it wouldn't make more sense to instead of changing the default partition size to let the installer make the partition as large as possible. Then you don't need to set a custom size.

Reply:
Certainly a possibility for another PR.